tests/mir-opt/nll/region_subtyping_basic.main.nll.0.32bit.mir

```diff
@@ -22,7 +22,7 @@
 |
 fn main() -> () {
     let mut _0: ();
-    let mut _1: [usize; Const { ty: usize, kind: Leaf(0x00000003) }];
+    let mut _1: [usize; ValTree(Leaf(0x00000003): usize)];
```

Comment:
This test seems to enable "always verbose" mode, that's why it doesn't get `3_usize`.